modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -30,6 +30,23 @@
 
 The following premium plugin updates were released alongside {productname} 6.5.
 
+=== Advanced Templates 1.2.0
+
+The {productname} 6.5 release includes an accompanying release of the **Advanced Templates** premium plugin.
+
+**Advanced Templates** 1.2.0 includes one fix:
+
+=== The currently selected tree component template item would not stay highlighted after the Templates dialog redialing.
+//#TINY-9770
+
+In previous versions of {productname} a selected tree component template item would lose its highlight when the Advanced Template dialog was reopened.
```

Comment:
```suggestion
In previous versions of {productname} a selected tree component template item would lose its highlight when the Advanced Template dialog was rerendered.
```

Reply:
@vpyshnenko Applied @brian-forte copy edit changes to this.